gender_novels/corpus_gen.py

```diff
@@ -0,0 +1,45 @@
+import csv
+import re
+from pathlib import Path
+
+from gender_novels import common
+from gender_novels import novel
+
+# @TODO: A lot of things
+
+# Not sure how things are going to work exactly, functions might not look like this in the end
+
+def get_novel_text(novel_id):
+    """
+    For a given novel id returns the full text of that novel as a string
+    @TODO: implement this function
+    """
+    pass
+
+def get_publication_date(novel_id):
+    """
+    For a given novel with id novel_id this function attempts a variety of methods to try and find the publication date
+    :param novel_id: int
+    :return: int
+    @TODO: implement this function
+    """
+    pass
+
+def get_publication_date_from_copyright(novel_text):
+    """
+    Tries to extract the publication date from the copyright statement in the given text
+
+    >>> get_publication_date_from_copyright("This work blah blah blah asldflkj COPYRIGHT, 1894 adfajdhflkjhalskdjfhlkjasdhf and they all died.")
```

Comment:
This test would be best if we can take in a sample from our corpus. Otherwise, if it could be a cleaner looking string that mimics the Gutenberg texts, that could be another solution. I think having an incoherent string is not the best for non-native English speakers and those who rely on screen readers. Making it more elegant looking would also be a plus.